src/plot.jl

```diff
@@ -12,7 +12,7 @@ end
 function to plot the states present in a density matrix output by QuantumAnnealing.simulate
 kwargs are for Plots.bar
 """
-function plot_states(ρ; order=:numeric, spin_comp=ones(Int(log2(size(ρ)[1]))), num_states=16, ising_model=nothing, energy_levels=0, kwargs...)
+function plot_states(ρ; order=:numeric, spin_comp=ones(Int(log2(size(ρ)[1]))), num_states=0, ising_model=nothing, energy_levels=0, kwargs...)
```

Comment:
Does it make more sense to default to using all states, or would it be better to change it to be both `num_states = 0` and `energy_levels = 1` since the more likely default use case for people would be plotting ground states?  That could prevent over-saturation on the x-axis for models larger than 4 qubits while also not providing an annoying limitation that only views low probability high energy states.

Reply:
I agree that `energy_levels=1` is probably what folks would like to see most of the time.  However, that requires that you also specify the `ising_model` argument, which is kind of tedious when you just want to have a quick look at what is going on. I don't think it can be the default behavior for that reason.

Reply:
I agree with that.  In that case, LGTM!